extensions/wikia/SpecialHealthcheck/HealthController.class.php

```diff
@@ -165,6 +162,34 @@ private function testHost( $databaseName, LoadBalancer $loadBalancer, $index ) {
 			return false;
 		}
 
+		return true;
+	}
+
+	/**
+	 * Execute checks for a single database server
+	 * @param LoadBalancer $loadBalancer Load Balancer instance for the given cluster
+	 * @param int $index Server index to test
+	 * @return bool Is server healthy?
+	 * @throws MWException
+	 */
+	private function testHost(LoadBalancer $loadBalancer, $index ) {
+		// connection check
+		try {
+			$db = wfGetDB( $index, array() );
```

Comment:
@Grunny Database which we connect to doesn't matter as far as we are connecting to a proper server. In this function we only gather server variables and server information using `getServerInfo()` I doubt that choosing different database will matter in this case.